diff --git a/node_modules/partykit/dist/bin.mjs b/node_modules/partykit/dist/bin.mjs
index fc339dc..6850dfb 100755
--- a/node_modules/partykit/dist/bin.mjs
+++ b/node_modules/partykit/dist/bin.mjs
@@ -7,6 +7,13 @@ import assert2 from "assert";
 import * as fs2 from "fs";
 import chalk2 from "chalk";
 
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+
 // src/fetchResult.ts
 import assert from "assert";
 import { fetch } from "undici";
